### PR TITLE
properly invert overview icons on collegeboard apclassroom

### PR DESCRIPTION
**Pull request by isaec**
before and afters below - not sure if this page can be easily accessed.

image

image

image

image

### CODE REVIEW

**Comment by isaec** [comment]
with the better selector, the magnify icon now looks like:
image

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -747,6 +747,14 @@ IGNORE IMAGE ANALYSIS
 
 ================================
 
+apclassroom.collegeboard.org
+
+INVERT
+a.overview-icon svg
+button.Zoom--expander svg g > *:first-child
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:754]:
```suggestion
button.Zoom--expander svg g > :first-child
```
In the world of CSS no need to specify the `*` in this case.

**Review by Gusted — APPROVED** [review]
Thank you!

**Comment by isaec** [comment]
yep, thank you! Is that sort of selector the right way to only select the first path of an svg otherwise?

**Comment by Gusted** [comment]
> yep, thank you! Is that sort of selector the right way to only select the first path of an svg otherwise?

Yup! That's the correct way to specify that.

**Comment by isaec** [comment]
> > yep, thank you! Is that sort of selector the right way to only select the first path of an svg otherwise?
> 
> Yup! That's the correct way to specify that.

got it, thanks again - i'm pretty new to webdev.

**Pull request merged**